README.md

```diff
@@ -195,24 +195,38 @@ Run `yarn license-report` to generate a license report for all npm packages. See
 
 ## DUS Modes:
 
-### Classic Mode
+### 1 Search-Mode
 
-This is first release of the DUS solution. The major services included in this mode include Amazon Elasticsearch, Amazon Textract and Amazon Comprehend that allow digitization, information extraction and indexing in DUS.
+DUS offers both Amazon Elasticsearch and Amazon Kendra as two options for indexing and searching data across documents
 
-### Comprehend Medical Enabled Mode
-In the Comprehend Medical Enabled mode, DUS provides the additional capability to extract medical information such as Medical Entities, ICD-10 and RX Norm onotlogies from uploaded documents.
-To enable this mode, set the ` enableComprehendMedical: "true"` in package.json
+#### 1.1. Amazon Elasticsearch Only
+
+In this version, DUS supports searching/indexing of documents using Amazon Elasticsearch. The document data and metadata are indexed in the Amazon ES cluster to provide the traditional search experience.
 
-### Kendra-Enabled Mode
+For Development Deploy, Set `searchMode` = `AMAZON_ES_ONLY` in package.json  
+For CICD Deploy, set the SearchMode parameter in the CloudFormation template.
 
-In the Classic version, DUS supports searching/indexing of documents using Amazon Elasticsearch
-In the kendra enabled mode, Amazon Kendra is added as an additional capability and can be used for exploring features such as Semantic Search, Adding FAQs and Access Control Lists. 
-For demo purposes, this mode comes pre-loaded with a sample dataset of medical documents. 
+#### 1.2. Amazon Kendra Only 
+
+In the Amazon Kendra enabled mode, Amazon Kendra is can be used for exploring features such as Semantic Search, Adding FAQs and User Context Based Filtering.
+
+For Development Deploy, Set `searchMode` = `AMAZON_KENDRA_ONLY` in package.json  
```

Comment:
NIT: can we say Set `searchMode: AMAZON_KENDRA_ONLY` in package.json? Just like we say for enableComprehendMedical and readonly modes.

Reply:
Updated